apps/src/lib/util/mlApi.js

```diff
@@ -0,0 +1,17 @@
+export const commands = {
+  async getPrediction(opts) {
+    console.log('this is where the ML code will live!');
+  }
+};
+
+export default function MLApi() {}
+
+MLApi.getModelNames = function() {
```

Comment:
This is where things fall apart because the names are back from the call in time to show in the block dropdown -- dead end or am I missing a better way to do something similar?

Reply:
<img width="492" alt="Screen Shot 2021-01-06 at 3 26 57 PM" src="https://user-images.githubusercontent.com/12300669/103817183-2be6ce80-5034-11eb-9237-fb1717b9c5bd.png">
This i what I wanted it to do.